### ISSUE TITLE
[R] The link between the number of rounds and the number of trees

### ISSUE THREAD

**Issue by YuriGagarine** [dmlc/xgboost#2041]
Hello everyone,

I am using XGBoost (R package) and I have a question about the link between the number of rounds and the number of trees. 

I am doing multiclass classification (objective function = 'multi:softmax') and I realized that :
number of trees = nrounds * nclasses

What I want is : number of trees = nrounds

My question is : Is it possible to have what I described ?

Thank you in advance,

Chakib

**Comment by pommedeterresautee** [comment]
I think you are thinking to random forest where trees are grow in parallel.
In boosting, trees are grown one after the other.
FYI You can simulate random forest in XGboost

I close the issue because it is an old one.
Don t hesitate to reopen it if required.
